src/composites/ConfigurationContainer/ImportanceAdjustment/ProfileSelection.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import { Box, Select, Button, Flex, Text, Separator } from "@radix-ui/themes";
+import { ChevronDownIcon, UploadIcon, SliderIcon, DropdownMenuIcon } from '@radix-ui/react-icons';
+import { Profile } from '../../../types';
+import profilesData from "../../../assets/PIQUE_json_files/ExampleProfile.json";
+
+interface ProfileSelectionProps {
+    onProfileChange: (selectedProfiles: Profile[] | null) => void;
+}
+
+export const ProfileSelection: React.FC<ProfileSelectionProps> = ({ onProfileChange }) => {
+    // Use state to manage the dynamically uploaded or customized profiles
+    const [selectedProfileType, setSelectedProfileType] = useState<string | null>(null);
+    const [uploadedProfiles, setUploadedProfiles] = useState<Profile[]>([]);
+
+    // Function to handle profile selection changes
+    const handleProfileChange = (value: string) => {
+        setSelectedProfileType(value);
```

Comment:
typically, we don't want to maintain state locally(`setSelectedProfileType`) but also in the parent component (`onProfileChange`). You probably want to use a state on the parent component for this info, and this component should just receive those props, and alert the parent on user interactions.

Reply:
addressed, rn state setSelectedProfile is set in the parent component, and the selectedProfile is passed to the child component (ProfileSelection)